.github/workflows/test.yml

```diff
@@ -80,33 +85,36 @@ jobs:
     strategy:
       matrix:
         include:
+          # Minimum supported rust (MSRV)
           - container:
-              image: georust/proj-ci:rust-1.53
+              image: georust/proj-ci:proj-9.0.0-rust-1.52
```

Comment:
I'm still waffling on if it's better to have explicit immutablish docker tags, or if it's better to treat tags like release branches and just clobber it knowing that it (should be) backwards compatible. 🤷